frappe/utils/csvutils.py

```diff
@@ -49,15 +50,33 @@ def read_csv_content(fcontent):
 
 		if not decoded:
 			frappe.msgprint(
-				_("Unknown file encoding. Tried utf-8, windows-1250, windows-1252."), raise_exception=True
+				_("Unknown file encoding. Tried utf-8-sig, utf-8, windows-1250, windows-1252."),
```

Comment:
This can use the list too then instead of needing to be manually populated

Reply:
@akhilnarang If it's generated string, I'm a bit worried that it might conflict with the _() localization/translation.
I mean something like this:
```
		if not decoded:
			frappe.msgprint(
				_(f"Unknown file encoding. Tried {", ".join(FILE_ENCODING_OPTIONS)}."),
				raise_exception=True,
			)
	```